src/main/java/com/spotify/asyncdatastoreclient/DatastoreImpl.java

```diff
@@ -39,10 +39,8 @@
 import com.google.datastore.v1.RunQueryRequest;
 import com.google.datastore.v1.RunQueryResponse;
 import com.google.protobuf.ByteString;
-import com.ning.http.client.AsyncHttpClient;
-import com.ning.http.client.AsyncHttpClientConfig;
-import com.ning.http.client.Response;
-import com.ning.http.client.extra.ListenableFutureAdapter;
+import org.asynchttpclient.*;
```

Comment:
I'd like to avoid wildcard imports if that's ok? Could you change this to only import the things we need?

Reply:
Sure. Done.